.github/workflows/infrastructure-lint.yml

```
@@ -0,0 +1,351 @@
+      - name: 🔍 Alert rules validation
+        run: |
+          echo "🔍 Validating alert rules..."
+          
+          if [ -f "monitoring/alerts/alert-rules.yml" ]; then
+            # Install yamllint
+            pip3 install yamllint
+            yamllint monitoring/alerts/alert-rules.yml
+            
+            # Basic validation for Prometheus alert syntax
+            python3 -c "
+import yaml
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 204: could not find expected ':'

(syntax-check)